src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING enterprise.license;
+~~~
+~~~
+             enterprise.license
++-------------------------------------------+
+  crl-0-ChB1x...
+(1 row)
+~~~
+
+The license setting is also logged in the cockroach.log on the node where the command is run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+$ cat cockroach.log | grep license
+~~~
+~~~
+I171116 18:11:48.279604 1514 sql/event_log.go:102  [client=[::1]:56357,user=root,n1] Event: "set_cluster_setting", target: 0, info: {SettingName:enterprise.license Value:xxxxxxxxxxxx User:root}
+~~~
+
+## Monitor for license expiry
+
+You can monitor the time until your license expires in the following ways:
+
+1. [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}): The `seconds_until_enterprise_license_expiry` metric reports the number of seconds until the license on a cluster expires. It will report `0` if there is no license, and a negative number if the license has already expired. For more information, see [Monitoring and Alerting]({% link {{ page.version.version }}/monitoring-and-alerting.md %}).
+1. [DB console]({% link {{ page.version.version }}/ui-overview.md %}): Several [license expiration messages]({% link {{ page.version.version }}/ui-overview.md %}#license-expiration-message) may be displayed, depending on the status of your cluster's license.
+
+## Renew an expired license
+
+To renew an expired **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a> and then [set](#set-a-license) the new license.
+
+To renew an expired **Enterprise Free** license, follow the steps in [Obtain a license](#obtain-a-license).
+
+## FAQs
+
+### Can I host CockroachDB as a service for internal use at my organization?
+
+Yes, employees and contractors can use your internal CockroachDB instance as a service, but no people outside of your organization will be able to use it without purchasing a license. Use of Enterprise features will always require a license.
+
+### What constitutes hosting CockroachDB as a service?
+
+Hosting CockroachDB as a service means creating an offering that allows third parties (other than your employees and contractors) to operate a database. Specifically, third parties cannot modify table schemas.
+
+<a name="throttling"></a>
+
+### What is throttling and how does it work?
+
+When a cluster is being throttled, the number of concurrent open [SQL transactions]({% link {{ page.version.version }}/transactions.md %}) is limited to 5.
+
+This will only happen in the following cases:
+
+- The cluster is not following telemetry requirements.
+    - There is a 7 day grace period for new **Enterprise Free** and **Enterprise Trial** clusters to start sending telemetry.
```

Comment:
This shouldn't be "new" clusters only. Any time there is an interruption in telemetry, the grace period should start.

Reply:
Updated a bit to the following, PTAL
----
    - There is a 7 day grace period for **Enterprise Free** and **Enterprise Trial** clusters to (re)start sending telemetry. This applies when a cluster is newly created, or any time there is an interruption in telemetry.